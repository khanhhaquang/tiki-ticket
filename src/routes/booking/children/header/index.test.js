import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from './index';

describe('Test Booking/Header component', () => {
	test('Snapshot header', () => {
		const component = renderer.create(<Header />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('Check header content not empty', () => {
		const component = shallow(<Header />);
		expect(component.find('.movie-name')).toBeTruthy();
		expect(component.find('.movie-detail')).toBeTruthy();
	});
});
