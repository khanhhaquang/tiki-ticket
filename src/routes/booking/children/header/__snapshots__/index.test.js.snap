// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Booking/Header component Snapshot header 1`] = `
<header
  className="sc-bdVaJa bJCXco header"
>
  <span
    className="movie-name"
  >
    Spider-man: Người nhện xa nhà
  </span>
  <span
    className="movie-detail"
  >
    C13 | 2D Vietsub
  </span>
</header>
`;
